docs/configuration.md

```diff
@@ -52,15 +52,25 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.delete.format.default        | data file format   | Default delete file format for the table; parquet, avro, or orc |
 | write.parquet.row-group-size-bytes | 134217728 (128 MB) | Parquet row group size                             |
 | write.parquet.page-size-bytes      | 1048576 (1 MB)     | Parquet page size                                  |
+| write.delete.parquet.page-size-bytes | 1048576 (1 MB)   | Parquet page size                                  |
 | write.parquet.dict-size-bytes      | 2097152 (2 MB)     | Parquet dictionary page size                       |
+| write.delete.parquet.dict-size-bytes | 2097152 (2 MB)   | Parquet dictionary page size                       |
 | write.parquet.compression-codec    | gzip               | Parquet compression codec: zstd, brotli, lz4, gzip, snappy, uncompressed |
 | write.parquet.compression-level    | null               | Parquet compression level                          |
+| write.delete.parquet.compression-codec | gzip           | Parquet compression codec: zstd, brotli, lz4, gzip, snappy, uncompressed |
+| write.delete.parquet.compression-level | null           | Parquet compression level                          |
 | write.avro.compression-codec       | gzip               | Avro compression codec: gzip(deflate with 9 level), zstd, snappy, uncompressed |
 | write.avro.compression-level       | null               | Avro compression level                             |
+| write.delete.avro.compression-codec | gzip              | Avro compression codec: gzip(deflate with 9 level), zstd, snappy, uncompressed |
```

Comment:
Clauses within parens should be separated from the preceding words by a space: `gzip (deflate ...)`.

Reply:
I have modified it.